@uportal/dashboard-carousel/src/components/DashboardCarousel.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div>
+    <template v-if="!dashboard">
+      <slot name="empty">
+        <p>no results found</p>
+      </slot>
+    </template>
+
+    <template v-else>
+      <Slick
+        ref="slick"
+        :options="slickOptions"
+        @afterChange="handleAfterChange"
+      >
+        <div
+          class="slick-item"
+          v-for="region of dashboard.folders"
+          :key="region.name"
+        >
+          <span
+            v-for="card of region.content"
+            :key="card.name"
+          >
+            <PortletRenderer :portlet-html-url="card.url" />
+          </span>
+        </div>
+      </Slick>
+      <ul ref="buttons">
+        <li
+          v-for="(region, index) in dashboard.folders"
+          :key="region.name"
+          :id="'dashboardCarousel-' + index"
+        >
+          <button
+            :class="{ 'active': activeIndex === index }"
+            class="btn"
+            @click="clickHandler(index)"
+          >
+            {{ region.name }}
+          </button>
+        </li>
+      </ul>
+    </template>
+  </div>
+</template>
+
+<script>
+import Slick from 'vue-slick';
+import PortletRenderer from './PortletRenderer';
+import Vue from 'vue';
+import AsyncComputed from 'vue-async-computed';
+import ky from 'ky';
+import oidc from '@uportal/open-id-connect';
+
+Vue.use(AsyncComputed);
+
+export default {
+  name: 'DashboardCarousel',
+  data: function() {
+    return {
+      slickOptions: {
+        dots: false,
+        arrows: true,
+      },
+      activeIndex: 0,
+    };
+  },
+  components: {
+    Slick,
+    PortletRenderer,
+  },
+  methods: {
+    clickHandler(slideIndex) {
+      this.activeIndex = slideIndex;
+      this.$refs.slick.goTo(slideIndex);
+    },
+    handleAfterChange(event, slick, currentSlide) {
+      this.activeIndex = currentSlide;
+    },
+  },
+  props: {
+    layoutApiUrl: {
+      type: String,
+      default: '/uPortal/api/v4-3/dlm/layout.json',
+    },
+  },
+  asyncComputed: {
+    layout: {
+      async get() {
+        const {layoutApiUrl, debug} = this;
```

Comment:
```
90:30  error    'debug' is assigned a value but never used  no-unused-vars
```

debug should be used on line 92 to replace `true`.
As well as included in `prop` on lines 81-86.

Reply:
Fixed